.github/workflows/docs.yml

```
@@ -0,0 +1,49 @@
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '18'
+      - uses: pnpm/action-setup@v2.4.0
+        name: Install pnpm
+        id: pnpm-install
+        with:
+          version: 8
+          run_install: false
+      - name: Get pnpm version
+        id: pnpm-version
+        run: |
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2005:style:1:6: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'

(shellcheck)